plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetReaderConfig.java

```diff
@@ -98,6 +98,19 @@ public boolean isUseColumnIndex()
         return options.isUseColumnIndex();
     }
 
+    @Config("parquet.use-bloom-filter")
+    @ConfigDescription("Enable using Parquet bloom filter")
+    public ParquetReaderConfig setUseBloomFilter(boolean useBloomFilter)
+    {
+        options = options.withUseBloomFilterIndex(useBloomFilter);
+        return this;
+    }
+
+    public boolean isUseBloomFilter()
+    {
+        return options.isUseBloomFilterIndex();
```

Comment:
Let's either use `index` suffix everywhere or drop it everywhere for consistency